frontend/src/action-creator.async.js

```diff
@@ -914,3 +916,69 @@ export const getUserMessagesSummary = userId => dispatch => {
     dispatch
   })
 }
+
+export const getAccessibleWorkspaces = userId => dispatch => {
+  return fetchWrapper({
+    url: `${FETCH_CONFIG.apiUrl}/users/${userId}/accessible_workspaces`,
+    param: {
+      credentials: 'include',
+      headers: {
+        ...FETCH_CONFIG.headers
+      },
+      method: 'GET'
+    },
+    actionName: ACCESSIBLE_WORKSPACE_LIST,
+    dispatch
+  })
+}
+
+export const getWorkspaceSubscriptions = userId => dispatch => {
+  return fetchWrapper({
+    url: `${FETCH_CONFIG.apiUrl}/users/${userId}/workspace_subscriptions`,
+    param: {
+      credentials: 'include',
+      headers: {
+        ...FETCH_CONFIG.headers
+      },
+      method: 'GET'
+    },
+    actionName: WORKSPACE_SUBSCRIPTION_LIST,
+    dispatch
+  })
+}
+
+export const joinWorkspace = (workspaceId, userId) => dispatch => {
```

Comment:
I have a slight preference for making it clear in the name of the function what method it is implementing, for example `postSpaceMember` or `postUserSpace`, but if you prefer to leave it like this, you can just solve this conversation. If you decide to change the name, I suggest you also change it to the `subscribeToWorkspace` function (maybe `postSpaceRequest`).

Reply:
This is indeed how the other functions are named in these files, the HTTP method appears in the name.